content/configuration.md

```diff
@@ -36,7 +35,7 @@ java cucumber.api.cli.Main
 
 {{% block "javascript" %}}
 ### CLI
-Cucumber.js includes an executable file to run the features. After installing Cucumber in your project, you can run it with:
+Cucumber.js includes an executable file to run the features. After installing Cucumber in project, you can run it with:
```

Comment:
Perhaps "in project" is a little bare?

Reply:
How does `in a project` sound? You hardly see `you\your` in docs.

Reply:
We quite often use you/your these days, in more informal docs written in active tense rather than passive.